megazords/full/android/build.gradle

```diff
@@ -1,7 +1,6 @@
 apply plugin: 'com.android.library'
 apply plugin: 'org.mozilla.rust-android-gradle.rust-android'
 apply plugin: 'kotlin-android'
-apply plugin: 'kotlin-android-extensions'
```

Comment:
This plugin reports being deprecated, and removing it didn't seem to break anything, so...